src/API/Helix/User/HelixFollow.ts

```diff
@@ -0,0 +1,35 @@
+import Twitch from '../../../';
+import { NonEnumerable } from '../../../Toolkit/Decorators';
+import HelixPagination from '../HelixPagination';
+import { UserIdResolvable } from '../../../Toolkit/UserTools';
+
+export interface HelixFollowFilter extends HelixPagination {
+	from?: UserIdResolvable;
+	to?: UserIdResolvable;
+}
+
+export interface HelixFollowData {
+    from_id: string;
+    to_id: string;
+    followed_at: string;
+}
+
+export default class HelixFollow {
+	@NonEnumerable protected _client: Twitch;
+
+    constructor(protected _data: HelixFollowData, client: Twitch) {
+        this._client = client;
+    }
+
+    get followedAt() {
+        return Date.parse(this._data.followed_at);
```

Comment:
For consistency with the rest of the library, please use `new Date()` instead of `Date.parse()` to return a `Date` object rather than a millisecond epoch timestamp.

Reply:
Then the Events one should also be updated.